dev-lang/gnucobol/gnucobol-3.0_rc1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="${P/_/-}"
+
+DESCRIPTION="an open-source COBOL compiler"
+HOMEPAGE="https://open-cobol.sourceforge.io"
+SRC_URI="mirror://sourceforge/open-cobol/${MY_P}.tar.gz"
+
+LICENSE="GPL-3+ LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="berkdb nls"
+
+RDEPEND="dev-libs/gmp:0
+	sys-libs/ncurses:0"
+DEPEND="${RDEPEND}
+	berkdb? ( sys-libs/db:4.8 )
```

Comment:
Are you sure it doesn't work with later ones? Upstream files don't indicate that anywhere. Also looking at my build.log it seems to be linking against 6.0 even though 4.8.30-r3 is installed. So do something like `>=sys-libs/db-4.1:*` here.